crates/cli/src/bin/wasm-bindgen-test-runner/deno.rs

```diff
@@ -18,19 +18,16 @@ pub fn execute(
 
         {console_override}
 
-        // global.__wbg_test_invoke = f => f();
+        window.__wbg_test_invoke = f => f();
 
         // Forward runtime arguments. These arguments are also arguments to the
         // `wasm-bindgen-test-runner` which forwards them to deno which we
         // forward to the test harness. this is basically only used for test
         // filters for now.
-        cx.args(Deno.args.slice(1));
-
-        const ok = await cx.run(tests.map(n => wasm.__wasm[n]));
-        if (!ok) Deno.exit(1);
+        cx.args(Deno.args);
 
         const tests = [];
-    "#,
+"#,
```

Comment:
Why the space removal there?

Reply:
The spacing isn't necessary for the generated code, so it gets cleaner, but its a cosmetic issue nevertheless. Thanks!

Reply:
Would you mind removing it then?
The JS output isn't meant to be beautiful at the moment, that's a whole other can of worms.

Reply:
Sure, done.